src/data/zmod/basic.lean

```diff
@@ -73,9 +73,6 @@ def comm_semigroup (n : ℕ) : comm_semigroup (fin (n+1)) :=
 
 local attribute [instance] fin.add_comm_semigroup fin.comm_semigroup
 
-lemma val_add {n : ℕ} : ∀ a b : fin n, (a + b).val = (a.val + b.val) % n
-| ⟨_, _⟩ ⟨_, _⟩ := rfl
-
```

Comment:
Should some of the other `fin`-related lemmas be moved to `data.fin` as well?

Reply:
I've moved the trivial lemmas val_mul, one_val and zero_val that seem reasonably to go along with val_add. The rest look more specific to how fin is being used to set up zmod n.